presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestUnweightedDoubleReservoirSample.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.reservoirsample;
+
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestUnweightedDoubleReservoirSample
+{
+    @Test
+    public void testGetters()
+    {
+        final UnweightedDoubleReservoirSample sample =
+                new UnweightedDoubleReservoirSample(200);
+
+        assertEquals(sample.getMaxSamples(), 200);
+    }
+
+    @Test
+    public void testIllegalBucketCount()
```

Comment:
- Rename to testInvalidMaxSamples
- Use `expectedExceptions` and `expectedExceptionsMessageRegExp`

```
    @Test(expectedExceptions=IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Maximum number of samples must be positive")
    public void testInvalidMaxSamples()
    {
        new UnweightedDoubleReservoirSample(0);
    }
```

Reply:
Should it be `expectedExceptions=PrestoException.class`?

Reply:
@atavory Yes. (At the time of writing the code was throwing IllegalArgumentException.)